data/schema/schema_v1.json

```diff
@@ -476,12 +481,37 @@
                                                                     "max_length": {
                                                                         "type": "integer"
                                                                     },
+                                                                    "decimal_places": {
+                                                                        "type": "integer"
+                                                                    },
                                                                     "alias": {
                                                                         "type": "string"
                                                                     },
                                                                     "repeats": {
                                                                         "type": "boolean"
                                                                     },
+                                                                    "max_value": {
+                                                                        "type": "object",
+                                                                        "properties": {
+                                                                            "value": {
+                                                                                "type": "integer"
+                                                                            },
+                                                                            "answer_id": {
+                                                                                "type": "string"
+                                                                            }
+                                                                        }
```

Comment:
We should also be able to enforce the one of `value` or `answer_id` being present using the `anyOf` pattern used elsewhere in the schema.